Added license and main page README (#1)
FedericoBerlfein/RomanChromaticPSF: .ipynb_checkpoints/README-checkpoint.md

> # Chromatic Effects on PSF and Shear Measurement for Roman

Reviewer: I put a whole load of comments here, but I just realized this is a checkpoint file?  Why is the readme in here?  My comments still apply but they should be used to change the README.md in the repo, not this file

## Introduction

> The software and data used are based on the paper:

Reviewer: ```suggestion
The software and data were used for the research that is described in this paper:
```

 - Chromatic Effects on the PSF and Shear Measurement for the Roman Space Telescope High-Latitude Wide Area Survey; Berlfein et al. (2025), submitted - [ADS entry](https://ui.adsabs.harvard.edu/abs/2025arXiv250500093B/abstract)

This repository includes software to:
 - Generate Roman-like image simulations of the [OpenUniverse2024 catalog](https://irsa.ipac.caltech.edu/data/theory/openuniverse2024/overview.html) or the [cosmoDC2 catalog](https://portal.nersc.gov/project/lsst/cosmoDC2/_README.html) using [GalSim](https://github.com/GalSim-developers/GalSim).
 - Generate catalog-level magnitude noise using [PhotErr](https://github.com/jfcrenshaw/photerr).
 - Perform shear measurement on galaxy images using [AnaCal](https://github.com/mr-superonion/AnaCal).

>  - Correct for SED-dependant chromatic effects on the PSF.

Reviewer: ```suggestion
 - Correct for SED-dependent chromatic effects on the PSF.
```

> Note that in order to generate the actual image simulations, the relevant extragalactic and stellar catalogs are not provided here and must be downloaded. We provide catalog-level magnitudes (both true and observed), along with their errors, for the Roman (YJHFW) and LSST (ugrizy) bands. All entries contain object IDs in case you wish to cross-match with the full simulated catalogs. 

Reviewer: The paragraph seems to say we don't provide the catalogs, and then that we do provide them.  I think you need a clarification somewhere that we provide the information you mentioned for a **subset** of the catalogs.

> ## Installation

Reviewer: Installation is not the same as cloning.  When we talk about installation, usually that's something that is done with a library, e.g., using pip, that results in an ability to import the library.  I would suggest changing this heading to something like "Using this software" and have an explanatory sentence that there is nothing to install, but rather a set of notebooks and scripts that people can run after cloning the repo.

You can clone the repository using:
```
git clone https://github.com/FedericoBerlfein/RomanChromaticPSF.git
```

## Guide to repository contents

> Below is an overview on the contents of this repository. More details on each folder can be found in each folders respective README. 

Reviewer: ```suggestion
Below is an overview of the contents of this repository. More details on each folder can be found in each folder's respective README. 
```

> - ``Gal_props_catnoise``: This folder contains the catalog-level noisy and true magnitudes, magnitude errors, object IDs, and redshift, for every object used in the paper. More details on the contents of this folder can be found in it's respective README.

Reviewer: ```suggestion
- ``Gal_props_catnoise``: This folder contains the catalog-level noisy and true magnitudes, magnitude errors, object IDs, and redshift, for every object used in the paper. More details on the contents of this folder can be found in its respective README.
```

- ``PaperFigures``: All the figures used in the paper.
- ``SED_fit_coeff``: The scripts used to fit the SEDs, as well as the results of the fits.
- ``gen_sims``: The scripts used to generate the image simulations and do the shape measurement.
- ``notebooks``: Some of the notebooks needed to either generate some of the intermediate data, analyze results, and plot relevant figures.
- ``shear_measurement_cosmoDC2/diffsky``: The measured shapes from the image simulations for different PSFs or correction schemes.
- ``sim_images``: Empty directory given that simulated images are too heavy to upload to Github.  Nevertheless, this is where there simulated images would live after running relevant scripts in ``gen_sims``.

## Contact us

In case you need help or have questions about the code, feel free to contact Federico Berlfein (fberlfei "at" andrew.cmu.edu) or use the [issues](https://github.com/FedericoBerlfein/RomanChromaticPSF/issues) on this repository.

## License

The code has been publicly released; it is available under the terms of our [LICENSE](LICENSE).

> If you make use of the software, please cite paper listed at the top of this README, and provide a link to this code repository.

Reviewer: ```suggestion
If you make use of the software, please cite the paper listed at the top of this README, and provide a link to this code repository.
```
